networking_generic_switch/config.py

```diff
@@ -30,6 +30,16 @@
 
 CONF.register_opts(coordination_opts, group='ngs_coordination')
 
+threadpool_opts = [
+    cfg.IntOpt('size',
+               min=1,
+               default=100,
```

Comment:
Would it better to have a default based on the number for cores?

Reply:
I'm not sure. You can't do real multithreading in python, so I don't know if more cores helps.